DOCS-1830: Remap tensor names (#2583)
viamrobotics/docs: docs/ml/vision/mlmodel.md

{{< /tabs >}}

Click **Save config**.

## Tensor names

Both the `mlmodel` detector and classifier require that the input and output tensors defined by your ML model are named according to the following:

- For an `mlmodel` detector:
  - The _input tensor_ must be named `image`
  - The _output tensors_ must be named `location`, `category`, and `score`,
- For an `mlmodel` classifier:
  - The _input tensor_ must be named `image`
  - The _output tensor_ must be named `probability`

> If you [trained your ML model using the Viam app](/ml/train-model/), your `mlmodel` tensors should already be named in this fashion, and you can proceed to [test your detector or classifier](#test-your-detector-or-classifier).

Reviewer: ```suggestion
If you [trained your ML model using the Viam app](/ml/train-model/), your `mlmodel` tensors is already named in this fashion, and you can proceed to [test your detector or classifier](#test-your-detector-or-classifier).
```
